src/hotrod/impl/operations/AddClientListenerOperation.cpp

```diff
@@ -78,7 +78,11 @@ char AddClientListenerOperation::executeOperation(transport::Transport& transpor
 			{
 				uint8_t status = codec20.readPartialEventHeader(transport, params);
 				if (!HotRodConstants::isSuccess(status))
-					break;
+				{
+				    auto errLen = transport.readVInt();
+				    auto errMsg = transport.readBytes(errLen);
+				    throw HotRodClientException(std::string(errMsg.begin(), errMsg.end()));
```

Comment:
@mgencur there's a bug on how filter/converter parameter are passed to the server. I've updated the ticket with a new PR.